package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "node-jose",
-  "version": "1.1.3",
```

Comment:
We like to keep the version increments as independent commits/PRs.  Please revert this line for now.

Reply:
I've updated this part as requested.